rules.mk

```diff
@@ -20,6 +22,9 @@ $(BUILD)/$(BIN).uf2: $(BUILD)/$(BIN).bin
 	@echo UF2CONV $@
 	@$(UF2) $^ -co $@
 
+$(SUBMODULES):
+	git submodule update --init --recursive
```

Comment:
Removing the `--recursive` flag dramatically cuts down on the time required to pull down submodules (from minutes to seconds), and TinyUSB's submodules all seem to be about building its own tests and examples.